app/app/layout.tsx

```diff
@@ -0,0 +1,25 @@
+import "../styles/global.scss";
+import AuthProvider from "@/contexts/Auth";
+import SidebarLayout from "@/components/SideBarLayout";
+
+/**
+ * The root layout for the whole app
+ * 
+ * todo: metadata
+ */
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    // <Html lang="en" style={{fontSize: "14px"}} data-bs-theme="dark">
```

Comment:
i left this here bc the dark them is glorious to dev on. would be fun to spin off an issue to make a UI toggle for it 🌚